### ISSUE TITLE
Linebreak before opening brace in multiline `if`

### ISSUE THREAD

**Issue by sfktrifork** [nicklockwood/SwiftFormat#475]
I'd love to see a rule controlling the placement of the opening brace in multiline `if` statements so that they better align with multiline `guard` statements, even when using K&R-style braces.

# My preference

I would write this...

```swift
if foo, bar, baz {
  print("all true")
}
```

...as...

```swift
if
  foo,
  bar,
  baz
{
  print("all true")
}
```

This is congruent with multiline `guard` statements (see #37).

```swift
guard
  foo,
  bar,
  baz
else {
  print("all true")
  return
}
```

# SwiftFormat's preference

Currently, SwiftFormat does this:

```swift
if
  foo,
  bar,
  baz {
  print("all true")
}
```

I would like to be able to control this behavior with a rule and still retain K&R-style braces for non-multiline statements for congruency with multiline `guard`.

**Comment by sharplet** [comment]
Another case where this would help readability is generic declarations with `where` clauses:

```swift
func foo<Foos: Sequence>(_ foos: Foos) -> [Foos.Element]
  where Foos.Element: Comparable
{
  // body
}
```

**Comment by nicklockwood** [comment]
@sfktrifork this is now implemented in 0.45.0 and later. The rule that handles this is `wrapMultilineStatementBraces`, which is enabled by default.
